<ui version="4.0" >
 <class>ReportSelectorDialogUI</class>
 <widget class="QDialog" name="ReportSelectorDialogUI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Render Farm Reports</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>6</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="0" column="1" >
      <widget class="QDateTimeEdit" name="mStart" />
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Start:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>End:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QDateTimeEdit" name="mEnd" />
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="mReportList" />
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mGenerateButton" >
       <property name="text" >
        <string>Generate Report</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mExitButton" >
       <property name="text" >
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
